### DISCUSSION TITLE
免费的HEVC

### DISCUSSION THREAD

**Discussion by PiPiTcc** [Richasy/Bili.Uwp#613] [展示]
希望可以添加到：[哔哩-常见问题](https://github.com/Richasy/Bili.Uwp/wiki/%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98)

链接：[免费HEVC拓展官网](https://www.free-codecs.com/hevc_video_extension_download.htm)

![Snipaste_2021-11-08_16-12-15](https://user-images.githubusercontent.com/47528293/140706616-b83e9287-3302-47ef-84ee-d9bc420bc9a0.png)
![Snipaste_2021-11-08_16-12-40](https://user-images.githubusercontent.com/47528293/140706765-92912d9a-68ef-44f6-b0f2-66a4262cb0dd.png)

**Comment by hanxinnaisi** [comment]
非常good

**Comment by Richasy** [comment]
已经加入，感谢分享
